src/cascadia/TerminalSettingsEditor/Actions.idl

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import "EnumEntry.idl";
+
+namespace Microsoft.Terminal.Settings.Editor
+{
+    runtimeclass ActionsPageNavigationState
+    {
+        Microsoft.Terminal.Settings.Model.CascadiaSettings Settings;
```

Comment:
It seems like we only need `GlobalAppSettings` to get access to the list of commands. We should pass that in instead of the whole settings object.

Reply:
Sure, this was me being quick and dirty

Reply:
I suspect we will eventually need the whole thing. Just like Profiles needs Color Schemes.

_what we really need is a viewmodel._